x/ethbridge/types/msgs.go

```diff
@@ -671,6 +672,20 @@ func (msg MsgSetBlacklist) GetSignBytes() []byte {
 }
 
 func (msg *MsgSetBlacklist) ValidateBasic() error {
+	if msg.From == "" {
```

Comment:
Should we also check if it is valid bech32 address?
`sdk.AccAddressFromBech32(msg.From)`

Reply:
will add it.